test/passing/functor.ml

```diff
@@ -55,7 +55,7 @@ module M =
 module type Module_type_fail = sig
   include S
 
-  module F : functor (_ : T) -> sig end
+  module F (_ : T) : sig end
```

Comment:
This is a regression. This syntax is new from `4.08`, we should preserve the long form.

Reply:
done